src/components/Category.js

```diff
@@ -37,6 +37,7 @@ const Category = ({
                 tagKey={key}
                 projects={graphProjects}
                 year={year}
+                showBlurb={tag !== 'misc'}
```

Comment:
I choosed to remove the blurb in the case of the 'misc' catagory, since the message doesn't fit at all for this category.